orchestra/views.py

```diff
@@ -190,6 +193,33 @@ def submit_task_assignment(request):
         raise BadRequest(e)
 
 
+@json_view
+@login_required
+def time_entries(request):
+    worker = Worker.objects.get(user=request.user)
+    try:
+        if request.method == 'GET':
+            return time_entries_for_worker(worker,
+                                           task_id=request.GET.get('task-id'))
+        elif request.method == 'POST':
+            time_entry_data = json.loads(request.body.decode())
+            if 'task_id' not in time_entry_data:
```

Comment:
that's handled by the serializer. It's a bit of a hack - task_id really should be a get parameter. but i stick it in the body